tools/target-browser/field_selector.go

```diff
@@ -9,6 +9,7 @@ var BackupSelector = []string{
 	"spec.backupPlan.uid as BackupPlan UID",
 	"status.creationTimestamp as Start Time",
 	"status.completionTimestamp as End Time",
+	"spec.clusterBackupPlan.uid as ClusterBackupPlan UID",
```

Comment:
can you not use `BackupPlan UID` here instead of `ClusterBackupPlan UID`?

Reply:
Yes, We can use this, updated